### DISCUSSION TITLE
Best practices for your tool

### DISCUSSION THREAD

**Discussion by fperillo** [vmware/packer-examples-for-vsphere#761] [Help]
Hi, first of all thank you for this tool that can give a beginner like me a real quick start.

I was finally able to complete a build of Ubuntu 22.04 (I will write another post on my experience) and despite I read a lot of blogs/howto I'm a bit lost on how to go further.

Imagine several ubuntu 22.04 server VMs that have different software installed with ansible, but more importantly different static IPs and disk layouts. How should I organize my configuration files?

I see there are some PRs queued to give some more configurability, are you going to merge them ?

Thank you

**Comment by tenthirtyam** [comment]
These are items that are currently in my "backlog" that I maintain in my available time. I do plan to merge them once they are tested and stable.

**Comment by fperillo** [comment]
I studied a bit more during the weekend and tried to use the configuration steps in vSphere, unfortunately still at version 7. I was able to create a cloned VM with a new IP and added a .cfg file to have cloud-init correctly set ntp servers, but was not able to correctly set default dns search domain.

Anyway, I think I have better understood how all the pieces fit together. I should only understand what should be in the default image and what can wait for a ansible run.

Thank you
